project.php

```diff
@@ -0,0 +1,21 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header("Access-Control-Allow-Origin: *");
+require 'vendor/autoload.php';
+
+try {
+    $db = \ProjectManager\Services\DbConnector::connect();
+        if (isset($_GET['id']) && is_numeric($_GET['id'])) {
+            try {
+                $data = \ProjectManager\Hydrators\ProjectsHydrator::getProject($db, $_GET['id']);
+                echo \ProjectManager\Services\ConvertToJsonService::convert($data, \ProjectManager\Services\ConvertToJsonService::PROJECT_SUCCESS_MESSAGE);
+            } catch (Throwable $e) {
```

Comment:
what is this catching?

Reply:
This handles if the project number is numeric but the project isn't a number